README.md

```diff
@@ -4,39 +4,64 @@ This repository is used for distributing kotlin releases of the Matrix Rust SDK.
 
 ## Releasing
 
+There is a ['Release SDK Library (parallel)'](https://github.com/matrix-org/matrix-rust-components-kotlin/actions/workflows/release_sdk_parallel.yml) Github Action you can run with a parameter for the SDK branch or SHA - tags don't work correctly at the moment - and the new version name to release.
```

Comment:
Could you mention that there is a different workflow for the crypto lib?

(Also, note that technically it is a "workflow", not an "action").